waspc/src/Wasp/AppSpec.hs

```diff
@@ -58,6 +58,10 @@ import qualified Wasp.SemanticVersion as SV
 -- describing the web app specification with all the details needed to generate it.
 -- It is standalone and de-coupled from other parts of the compiler and knows nothing about them,
 -- instead other parts are using it: Analyzer produces AppSpec while Generator consumes it.
+--
+-- IMPORTANT: Do not change this data structure without updating the AppSpec in
```

Comment:
I had this random 💩 idea, run a LLM on each commit if this files changed or the TS file changed. "Are these two things  the same implementation?"